uiowa/uiowa
# Make Article Captions Captiony Again

Resolves #3561 




# How to test

`git checkout article-captions && git pull && ddev blt frontend && ddev blt ds --site=default && ddev drush @default.local uli`
Create an article with a featured image and a featured image caption
Check things out!
Test various combos of featured image, featured image caption, and global caption (Check that the caption doesn't render when there is no featured image)
Test that global captions still apply to images added to a photo gallery
Test on some or all of the splits listed in https://github.com/uiowa/uiowa/pull/5993#issuecomment-1340115714
